saleor/order/tests/test_order_utils.py

```diff
@@ -306,7 +306,7 @@ def test_add_gift_cards_to_order(
 
     # when
     add_gift_cards_to_order(
-        checkout_info, order, Money(20, gift_card.currency), staff_user, None
+        checkout_info, order, Money(30, gift_card.currency), staff_user, None
```

Comment:
Why are we changing this?

Reply:
Because the test was checking wrong assumptions - there were two giftcards - with 10 and 20 balance, the test checked if both of the giftcards were zeroed after they were used even though on one card there should be 10 left. This would be valid when balance on both cards were equal to this number - that's why I raised it. 
And I've added the separate tet for case where two cards are above the total.